eo-runtime/src/test/eo/org/eolang/runtime-tests.eo

```diff
@@ -389,3 +389,20 @@
     value.@
     100
 
+[] > moves-copy-to-new-parent
+  assert-that > @
+    [] > app
+      [] > @
+        "Jeff" > x
+        QQ.txt.sprintf > @
+          "[%s]"
+          foo
+            $:^
```

Comment:
@yegor256 Can we use any object here? Like `^:^` or `QQ.io.stdout:^`. If so maybe it would be better to change `^` name to something else?

Reply:
@Graur yes, any object may stay at the left side of `:`, but only a name OR `^` on the right. We use `^` since it's our textual form of `\rho`

Reply:
@yegor256 Oh, it is a parent object. Now I get it